AssetLoader.cs

```diff
@@ -74,6 +79,37 @@ public static Texture2D LoadTexture(string name)
         return CreateDummyTexture();
     }
 
+    /// <summary>
+    /// Loads a animation with the specific name. If the animation isn't found from any
+    /// asset search path, returns a dummy animation with 1 frame.
+    /// </summary>
+    /// <param name="name">The name of the animation.</param>
+    /// <returns>The animation if it was found and could be loaded, otherwise a dummy animation.</returns>
+    public static Image LoadAnimation(string name)
```

Comment:
```suggestion
    public static Image LoadGIFAnimation(string name)
```

Currently this only supports loading GIFs, not other types of animations.